var/spack/repos/builtin/packages/rust-bindgen/package.py

```diff
@@ -2,28 +2,30 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-import os
-
 from spack.package import *
 
 
-class RustBindgen(Package):
+class RustBindgen(CargoPackage):
     """The rust programming language toolchain"""
 
     homepage = "https://www.rust-lang.org"
     url = "https://github.com/servo/rust-bindgen/archive/v0.20.5.tar.gz"
```

Comment:
```suggestion
    homepage = "https://rust-lang.github.io/rust-bindgen/"
    url = "https://github.com/rust-lang/rust-bindgen/archive/v0.20.5.tar.gz"
```

Two quick suggestions to update the `url` and `homepage` attributes to their newer values.

For url the repository was transferred to `rust-lang` and was currently redirecting there. This just cements that change and prevents an attack vector taking over the old url.